sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -307,6 +322,7 @@ class ApiManager {
     const xhrOptions: XhrOptions = {
       method: 'POST',
       successCode: 201,
+      failureCodes: [409],
```

Comment:
These really are failures though, but on the server side. The client should just be able to gracefully handle them.
Would naming it `possibleFailureCodes` be better?